plantcv/plantcv/image_fusion.py

```diff
@@ -0,0 +1,79 @@
+# Fuse two images
+
+import os
+import numpy as np
+from plantcv.plantcv import fatal_error
+from plantcv.plantcv import Spectral_data
+from plantcv.plantcv import params
+from plantcv.plantcv.hyperspectral.read_data import _make_pseudo_rgb
+from plantcv.plantcv._debug import _debug
+
+
+def image_fusion(img1, img2, wvs1, wvs2, array_type = None, filename = None):
+    """ Fuse two images of the same size together with given wavelengths representing and make a Spectral_data instance
+    img1: 1st image to be fused
+    img2: 2nd image to be fused
+    wvs1: list of wavelengths represent bands in img1
+    wvs2: list of wavelengths represent bands in img2
+    array_type: (optional) description of the fused array
+    filename: (optional) desired filename of the fused array
+
+    :param img1: np.ndarray
+    :param img2: np.ndarray
+    :param wvs1: list
+    :param wvs2: list
+    :param array_type: str
+    :param filename: str
+    :return: fused_array (a Spectral_data instance)
+    """
+
+    if len(img1.shape) == 2:
+        img1 = np.expand_dims(img1,axis=2)
+    r1, c1, b1 = img1.shape
+
+    if len(img2.shape) == 2:
+        img2 = np.expand_dims(img2,axis=2)
+    r2, c2, b2 = img2.shape
+    if (r1,c1) != (r2,c2):
+        fatal_error("Input images should have the same image size")
+
+    array_data  = np.concatenate((img1, img2), axis=2)
+
+    # sort all wavelengths
+    wavelengths = np.array(wvs1 + wvs2)
+    ind = np.argsort(wavelengths)
+    wavelengths = wavelengths[ind]
+
+    wavelength_dict = dict()
+    for (idx, wv) in enumerate(wavelengths):
+        wavelength_dict[wv] = float(idx)
+
+    # sort array_data based on wavelengths
+    array_data = array_data[:,:,ind]
```

Comment:
```suggestion
    array_data = array_data[:, :, ind]
```

Add missing spaces after commas